tools/chef/roles/db-mysql56-server.json

```diff
@@ -17,12 +33,14 @@
         "packages": [
           "Percona-Server-devel-56",
           "Percona-Server-server-56"
-        ]
+        ],
+        "service_name": "mysql"
```

Comment:
this is specific to Percona on RHEL/CentOs 6. On CentOS 7 it's mysqld (annoying, I know)